resources/views/partials/content-single-pcc-story.blade.php

```diff
@@ -0,0 +1,39 @@
+@php $org = get_post_meta (get_the_ID(), 'pcc_story_organization', true) @endphp
+<article @php post_class('container') @endphp>
+  <header>
+    @include('partials/breadcrumb')
+    <h1 class="entry-title">{{ $org }} - "{!! App::title() !!}"</h1>
+  </header>
+  @include('partials/entry-meta')
+
+  <div class="content" id="content">
+    <div class="pcc-story-video-container">
+    @php echo wp_oembed_get(get_post_meta (get_the_ID(), 'pcc_story_video_link', true)) @endphp
+    </div>
+    @content
+  </div>
+  <footer>
+    <div class="tags-container">
+      @php
+          $sectorList = SinglePccStory::sectors();
+          $regionList = SinglePccStory::regions();
+      @endphp
+
+      @if ($sectorList)
+      <p>{{ __('Sectors', 'pcc') }}</p>
+      {!! $sectorList !!}
+      @endif
+
+      @if ($regionList)
+      <p>{{ __('Regions', 'pcc') }}</p>
+      {!! $regionList !!}
+      @endif
```

Comment:
The assignment isn't needed; the same functionality can be obtained like this:

```php
@if ($sectors)
  // Code goes here.
@endif
```
The `sectors()` and `regions()` methods aren't static so they don't need to be called statically (and in fact shouldn't be); they are automatically assigned to snake-cased variables.